activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/CustomNamespaceAttributeConverterTest.java

```diff
@@ -19,12 +19,14 @@
   @Test
   public void convertXMLToModel() throws Exception {
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull("BPMN Model XML not found", bpmnModel);
     validateModel(bpmnModel);
   }
 
   @Test
   public void convertModelToXML() throws Exception {
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull("BPMN Model XML not found", bpmnModel);
     BpmnModel parsedModel = exportAndReadXMLFile(bpmnModel);
     validateModel(parsedModel);
     deployProcess(parsedModel);
```

Comment:
Is the deployProcess() method doing something? if not we should start getting rid of dead code.